torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12003,6 +12003,8 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
             # This is only failing on Linux Bionic 3.10 Cuda 11.6
             DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_dtypes',
                          device_type='cuda', active_if=_get_torch_cuda_version() >= (11, 6)),
+            DecorateInfo(unittest.skip("Skipped!"), 'TestCommon', 'test_noncontiguous_samples',
```

Comment:
This is only failing because this calls efficient_attention which can handle all non-contiguity but the last dim not ==1. So I added the skip here but also could add another gate so that we go down the math part. But I feel like should fail loudly in this case and someone
can only enable the math kernel